sample-apps/react/react-video-demo/index.html

```diff
@@ -1,41 +1,118 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="icon" type="image/png" sizes="192x192" href="/images/icons/android-icon-192x192.png">
-    <link rel="icon" type="image/png" sizes="256x256" href="/images/icons/android-icon-256x256.png">
-    <link rel="icon" type="image/png" sizes="384x384" href="/images/icons/android-icon-384x384.png">
-    <link rel="icon" type="image/png" sizes="512x512" href="/images/icons/android-icon-512x512.png">
-
-    <link rel="apple-touch-icon" sizes="57x57" href="/images/icons/apple-icon-57x57.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-60x60.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-72x72.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-76x76.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-114x114.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-152x152.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/icons/apple-icon-180x180.png">
+    <meta
+      name="apple-itunes-app"
+      content="app-id=1644313060, app-clip-bundle-id=io.getstream.iOS.VideoDemoApp.Clip, app-clip-display=card"
+    />
```

Comment:
This is the relevant change, everything else is prettier doing its formatting magic.